### PR TITLE
Updating Formatting

### PR DESCRIPTION
**Pull request by jasonpatrickellykrause**
- Corrected missing code fences
- Corrected list indentations
- Corrected a couple of typos
- Added/removed spacing
- Beautified a couple of tables
- Changed heading levels to be sequential

### CODE REVIEW

File: readarr/faq.md [hunk 1]
```diff
@@ -53,9 +55,10 @@ dateCreated: 2021-05-25T20:01:09.320Z
   - [How do I stop the browser from launching on startup?](#how-do-i-stop-the-browser-from-launching-on-startup)
   - [Weird UI Issues](#weird-ui-issues)
   - [VPNs, Jackett, and the \*ARRs](#vpns-jackett-and-the-arrs)
+    - [Use of a VPN](#use-of-a-vpn)
   - [Jackett's /all Endpoint](#jacketts-all-endpoint)
     - [Jackett /All Solutions](#jackett-all-solutions)
-  - [Why are there two files? | Why is there a file left in downloads?](#why-are-there-two-files-why-is-there-a-file-left-in-downloads)
+  - [Why are there two files? | Why is there a file left in downloads?](#why-are-there-two-files--why-is-there-a-file-left-in-downloads)
```
Comment (bakerboy448) [inline readarr/faq.md:61]: Need to manually fix these. Wikijs ignores the special chars `--` to `-`

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

File: readarr/faq.md (hunk 1 above)
Comment (jasonpatrickellykrause) [reply readarr/faq.md:61]: I should learn more about Wikijs. `:shame:`

Comment (bakerboy448) [reply readarr/faq.md:61]: Learn by doing (it was something I stumbled on as well)

File: radarr/faq.md [hunk 2]
```diff
@@ -58,23 +60,26 @@ dateCreated: 2021-05-16T20:44:27.778Z
   - [Weird UI Issues](#weird-ui-issues)
   - [Web Interface Only Loads at localhost on Windows](#web-interface-only-loads-at-localhost-on-windows)
   - [Permissions](#permissions)
-  - [System & Logs loads forever](#system-logs-loads-forever)
-  - [Finding Cookies](#finding-cookies)
+  - [System \& Logs loads forever](#system--logs-loads-forever)
```
Comment (bakerboy448) [inline radarr/faq.md:63]:
```suggestion
  - [System \& Logs loads forever](#system-logs-loads-forever)
```

**Pull request merged**